contracts/cw1-whitelist/src/contract.rs

```diff
@@ -62,6 +68,29 @@ impl Cw1WhitelistContract {
 
         Ok(Response::new())
     }
+
+    #[msg(exec)]
+    fn remove_member(
+        &self,
+        (deps, _env, msg): (DepsMut, Env, MessageInfo),
```

Comment:
`(deps, _env, msg): (DepsMut, Env, MessageInfo),`

This reads a bit ugly / log. Especially elsewhere where we have `cosmwasm_std::DepsMut`.
If we define some struct like `type CtxMut = (DepsMut, Env, MessageInfo)` that could make this read cleaner (and maybe fit on one line).

I do like the destructuring in the arg list